spec/test-outputs/www-integration.out.vcl

```diff
@@ -426,6 +454,32 @@ sub vcl_deliver {
   }
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
+  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker") {
+    declare local var.variant STRING;
+    set var.variant = req.http.GOVUK-ABTest-StartABusinessSegment;
+
+    declare local var.sab_expiry TIME;
+    if (var.variant == "C") {
+      # If they have opted out of the banner or have visited the SaB checker
+      # Expire in 26 weeks (approx 6 months)
+      set var.sab_expiry = time.add(now, std.integer2time(15724800));
```

Comment:
Is it also possible to use `time.add` as shown [in the docs](https://developer.fastly.com/reference/vcl/functions/date-and-time/time-add/) and specify the unit of time you want to add (avoiding a call to `std.integer2time()`?